packages/discord.js/typings/index.d.ts

```diff
@@ -508,23 +510,6 @@ export type BooleanCache<Cached extends CacheType> = Cached extends 'cached' ? t
 export abstract class CommandInteraction<Cached extends CacheType = CacheType> extends BaseInteraction<Cached> {
   public type: InteractionType.ApplicationCommand;
   public get command(): ApplicationCommand | ApplicationCommand<{ guild: GuildResolvable }> | null;
-  public options: Omit<
```

Comment:
Am I missing something? This was only added back to `ContextMenuCommandInteraction` but not `ChatInputCommandInteraction`

At which point, I'd also ask you extract it into a helper type.

Reply:
`ChatInputCommandInteraction` already had it defined, since it overwrote the type of it.

Reply:
Ahh, I had even looked at the full file but didn't realize. If it was properly `override` it would've showed up in this diff too and been more obvious.